### PR TITLE
Connect: Send ptp4l_id from Proxy to Client once ptp4l is connected

### PR DESCRIPTION
**Pull request by JunAnnLaiIntel**
1. Client is receiving ptp4l_id once proxy is connected to ptp4l
image

2. Client will getting connection failed if proxy is not connected to ptp4l
image

### CODE REVIEW

File: jclklib/client/connect_msg.cpp [hunk 1]
```diff
@@ -7,6 +7,8 @@
 
 #include <client/connect_msg.hpp>
 #include <common/print.hpp>
+#include <common/serialize.hpp>
+#include <client/subscribe_msg.hpp>
```
Comment (yoongsiang2) [inline jclklib/client/connect_msg.cpp:11]: why need to include subscribe msg header?

File: jclklib/proxy/connect.cpp [hunk 2]
```diff
@@ -228,6 +228,12 @@ int Connect::connect()
     }
     m_sk.reset(sku);
 
+    while (system("ls /var/run/ptp4l*") != 0) {
```
Comment (AzuraTarmiziIntel) [inline jclklib/proxy/connect.cpp:231]: For good code practice, do init the pe.ptp4l_id  to zero. 
Btw, i didnt see the pe structure being initialized to init values , before being used. 
Maybe we can have the init somewhere before  Connect in proxy started / or before this is used : ptp4l_event_loop

**Review by AzuraTarmiziIntel — CHANGES_REQUESTED** [review]

**Comment by AzuraTarmiziIntel** [comment]
The changes are good. Just additional init i requested.

File: jclklib/proxy/connect.cpp (hunk 2 above)
Comment (AzuraTarmiziIntel) [reply jclklib/proxy/connect.cpp:231]: JClkLibCommon::ptp_event pe = { 0, {0, 0, 0, 0, 0, 0, 0, 0}, 0 , 0 };

**Review by AzuraTarmiziIntel — APPROVED** [review]
LGTM. THanks for the changes

**Pull request merged**